### ISSUE TITLE
Invisible docked overlay menu

### ISSUE THREAD

**Issue by lajax01** [WerWolv/nx-ovlloader#6]
Latest release, with latest Atmosphere, on 10.0.1 fw.
In home screen work fine in docked, but ingame wont show up.
Its frose my controlls ingame, seems to me its there, but invisible.
If I press B (quit from Tesla normaly) I gain back controll.
Please advise.
In handheld just works as normaly would.
Im tested: Im entered Tesla ingame in handheld, then I put into the dock, permanently dissapear Tesla on TV, then I take it out, and the Tesla overlay is still there.. 
Something wrong with docked ingame overlay.
Please advise.

**Comment by deadair3210** [comment]
it's a scaling issue, set tv scaling to 75 or 100, don't remember which. either way, it's a known issue

**Comment by lajax01** [comment]
> 
> 
> it's a scaling issue, set tv scaling to 75 or 100, don't remember which. either way, it's a known issue

Please help a little bit. Im not a noob, but Im dont realy understand where to set this scaling what you mentioned.. :D

**Comment by TechGeekGamer** [comment]
It’s in the switch’s System Settings. You scroll down to TV settings and there is something about scaling

**Comment by TechGeekGamer** [comment]
> it's a scaling issue, set tv scaling to 75 or 100, don't remember which. either way, it's a known issue

It’s 100.

**Comment by lajax01** [comment]
> 
> 
> > it's a scaling issue, set tv scaling to 75 or 100, don't remember which. either way, it's a known issue
> 
> It’s 100.

Thanks a lot. If Im get to home, I will check this, then report back.

**Comment by lajax01** [comment]
Yes, that was it.. I used from ages back then I thing around 5.2 fw screen scalling 96%.
And with that, on 9.2 cfw worked out from box.
Now I had to use 100%, to see Overlays again.
But with this, my picture overgrown my screen size.. :D

**Comment by TeeToo2008** [comment]
> it's a scaling issue, set tv scaling to 75 or 100, don't remember which. either way, it's a known issue

you are my hero bro. thank u alot.
this problem was driving me crazy.
